server/executor/testrunner/testrunner_entities.go

```diff
@@ -0,0 +1,87 @@
+package testrunner
+
+import (
+	"github.com/kubeshop/tracetest/server/pkg/id"
+	"github.com/kubeshop/tracetest/server/resourcemanager"
+	"golang.org/x/exp/slices"
+)
+
+var (
+	RequiredGateAnalyzerScore RequiredGate = "analyzer-score"
+	RequiredGateAnalyzerRules RequiredGate = "analyzer-rules"
+	RequiredGateTestSpecs     RequiredGate = "test-specs"
+)
+
+const (
+	ResourceName       = "TestRunner"
+	ResourceNamePlural = "TestRunners"
+)
+
+var Operations = []resourcemanager.Operation{
+	resourcemanager.OperationGet,
+	resourcemanager.OperationList,
+	resourcemanager.OperationUpdate,
+}
```

Comment:
I think this is not being used. we should try to remove the references to resource manager from packages